pipeline/07-export.R

```diff
@@ -341,45 +397,25 @@ upload_data_prepped <- assessment_pin %>%
       select(meta_year, meta_pin, meta_card_num, meta_lline_num),
     by = c("meta_year", "meta_pin")
   ) %>%
-  mutate(meta_pin10 = str_sub(meta_pin, 1, 10)) %>%
-  group_by(meta_pin10, meta_tieback_proration_rate) %>%
```

Comment:
By my reading, the `group_by` statements are no longer necessary in this query, since we don't need to report `pred_pin10_final_fmv_bldg`; but I could be wrong!